UI/RemoteISOScreen.cpp

```diff
@@ -397,7 +398,7 @@ void RemoteISOScreen::CreateViews() {
 		rightColumnItems->Add(new Choice(sy->T("Share Games (Server)")))->OnClick.Handle(this, &RemoteISOScreen::HandleStartServer);
 		browseChoice->SetEnabled(true);
 	}
-	Choice *settingsChoice = new Choice(sy->T("Settings"));
+	Choice *settingsChoice = new Choice(pa->T("Settings"));
```

Comment:
Not that System was really right, but Pause doesn't seem related.  I hate reusing a word like "Settings" in different contexts... I guess it's hopefully the same here.

-[Unknown]